2020-07-23T18:57:15|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\ScopeFoundry\measurement.py", line 25, in run
    self.measurement._thread_run()
  File "C:\ProgramData\Anaconda3\lib\site-packages\ScopeFoundry\measurement.py", line 179, in _thread_run
    self.run()
  File "C:\LabPrograms\Python\DAQ_Laser_Camera\PROCHIP_Measurement_cell_detection.py", line 184, in run
    if self.settings.selected_channel.val == 0:
AttributeError: 'LQCollection' object has no attribute 'selected_channel'

